src/System.Private.Xml.Linq/tests/misc/LoadSaveAsyncTests.cs

```diff
@@ -26,6 +27,7 @@ public static void ArgumentValidation()
             Assert.Throws<ArgumentNullException>(() => { new XElement("Name").SaveAsync((XmlWriter)null, CancellationToken.None); });
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]   //Async methods defined only on Core
```

Comment:
The second argument to SkipOnTargetFramework is a string reason; that can be used instead of a comment.